<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>398</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>381</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="configLayout">
     <item>
      <widget class="QLabel" name="labelKBPaths">
       <property name="text">
        <string>KB Paths:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwKBPaths">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_3">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Watch Folders:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lwWatch">
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="themeLayout">
       <item>
        <widget class="QLabel" name="labelTheme">
         <property name="text">
          <string>UI Theme :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbTheme">
         <item>
          <property name="text">
           <string>System</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Light</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Dark</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="positionLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Launch Position :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cbLaunchPos">
         <item>
          <property name="text">
           <string>Left Center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Center</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Right Center</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="bbButtons">
       <property name="standardButtons">
        <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::RestoreDefaults|QDialogButtonBox::StandardButton::Save</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
